drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/importer.rb

```diff
@@ -201,6 +201,12 @@ def pre_process_class!(file_name, klass)
       end
     end
 
+    def self.expiring_models
+      importable_files.values.filter do |model|
+        model.column_names.include?('expired') && model.name.demodulize != 'Export'
```

Comment:
One more thought here.  I _think_ we should also keep Project.  The definition of the export is essentially "the details in Export.csv + the projects in Project.csv."  So keeping those records which get repeated in pretty much every import, would allow us to know if some data should or should not have been included in the import (within the last 5 of any given record.)
```suggestion
        # Don't expire or remove Export or Project records since they define the scope of a given import
        model.column_names.include?('expired') && ! model.name.demodulize.in?(['Export', 'Project'])
```